package.json

```diff
@@ -160,7 +164,7 @@
     "lint-staged": "^10.4.0",
     "lodash": "^4.17.15",
     "lodash.clonedeep": "^4.5.0",
```

Comment:
Being that `lodash.clonedeep` is now required for `formatInTimeZone|zonedTimeToUtc` this should either go in:

- `dependencies` so that `date-fns-tz` would ship with `lodash.clonedeep` packaged already
- `peerDependencies` to inform developers that to use `date-fns-tz` they will need to install this or `date-fns-tz` may not work if they use either of those functions.